cmake/CMakeLists.txt

```diff
@@ -4,39 +4,58 @@ cmake_minimum_required(VERSION 2.8)
 # Project
 project(protobuf C CXX)
 
+# CMake policies
+cmake_policy(SET CMP0022 NEW)
+
 # Options
+option(protobuf_VERBOSE "Enable for verbose output" OFF)
```

Comment:
I would be fine if all the options are named consistently (i.e., all without prefix or all with prefix). If project_OPTION_NAME is a common convention in cmake, using it is fine to me. I know that gtest uses the simple form though. Could you show me some references to projects using the protobuf_VERBOSE style naming?